manifests/mod/ssl.pp

```diff
@@ -11,6 +11,12 @@
     'freebsd' => '/var/run/ssl_scache(512000)',
   }
 
+  $stapling_cache = $::osfamily ? {
+    'debian'  => "\${APACHE_RUN_DIR}/ocsp(128000)",
+    'redhat'  => '/var/cache/mod_ssl/ocsp(128000)',
+    'freebsd' => '/var/run/ocsp(128000)',
```

Comment:
Hmmm. might i suggest we enable cache to be set as a variable that can be
set to desired value during config.

I think this then bypasses the need to set default

On 12 February 2015 at 21:24, Morgan Haskel <notifications@github.com>
wrote:

> In manifests/mod/ssl.pp
> <https://github.com/puppetlabs/puppetlabs-apache/pull/907#discussion_r24611320>
> :
>
> > @@ -11,6 +11,12 @@
> >      'freebsd' => '/var/run/ssl_scache(512000)',
> >    }
> >
> > +  $stapling_cache = $::osfamily ? {
> > +    'debian'  => "\${APACHE_RUN_DIR}/ocsp(128000)",
> > +    'redhat'  => '/var/cache/mod_ssl/ocsp(128000)',
> > +    'freebsd' => '/var/run/ocsp(128000)',
>
> @libb-city <https://github.com/libb-city> The issue is there's no default
> case for the selector.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-apache/pull/907/files#r24611320>
> .
>